src/Symfony/Component/Profiler/README.md

```diff
@@ -0,0 +1,45 @@
+Profiler Component
+==================
+
+Profiler collects information about each request made to your
+HttpKernel-based application and store them for later analysis.
```

Comment:
except if I'm wrong, the Profiler component don't depend of the HttpKernel component, correct ?